deploy/crds/pulpproject_v1alpha1_pulp_cr.travis.yaml

```diff
@@ -8,6 +8,7 @@ spec:
     access_mode: "ReadWriteOnce"
     # We have a little over 40GB free on Travis VMs/instances
     size: "40Gi"
+  pulp_settings:
```

Comment:
So in the default yaml, it's commented out. In this travis yaml, it is empty. Shouldn't we just remove this line? You do have `if pulp_settings is defined and pulp_settings is not none` in main.yaml, so it should be safe either way.